src/views/checkups/network/utils/utils.ts

```diff
@@ -44,6 +44,8 @@ export const STATUS_MIN_LATENCY_NANO = 'status.result.minLatencyNanoSec';
 export const STATUS_TARGET_NODE = 'status.result.targetNode';
 export const STATUS_SOURCE_NODE = 'status.result.sourceNode';
 
+export const CONFIG_PARAM_TARGET_NODE = 'spec.param.targetNode';
+export const CONFIG_PARAM_SOURCE_NODE = 'spec.param.sourceNode';
 export const STATUS_SAMPLE_DURATION = 'spec.param.sampleDurationSeconds';
```

Comment:
Please consider renaming the other spec parameters in a follow-up:
- STATUS_SAMPLE_DURATION
- STATUS_NAD_NAMESPACE
- STATUS_NAD_NAME
- STATUS_MAX_DESIRED_LATENCY

As they represent user defined configuration and not status reported by the checkup.

Reply:
done

Reply:
Thanks